sample/Kmd.Logic.Gateway.Automation.Sample/Kmd.Logic.Gateway.Automation.Sample.csproj

```diff
@@ -32,11 +26,4 @@
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </None>
   </ItemGroup>
-
-  <ItemGroup>
```

Comment:
Why this has been removed

Reply:
Why do we need this? It is just sample. This description should be on the dotnet tool nuget package.